tests/Console/ConfigurationResolverTest.php

```diff
@@ -1267,7 +1267,7 @@ public function provideDeprecatedFixerConfiguredCases()
     public function provideGetDirectoryCases()
     {
         return [
```

Comment:
can we add a cases like the following, to show that `basedir` is not messing up with single dot?
`php-cs-fixer fix .`
`php-cs-fixer fix src/File.php`

Reply:
So I think I can add the following test cases
```
            [null, './my/file', './my/file'],
            [null, 'my/file', 'my/file'],
            ['/my/path2/dir/.php-cs-fixer.cache', './dir2/file', './dir2/file'],
            ['/my/path2/dir/.php-cs-fixer.cache', 'dir2/file', 'dir2/file'],
            ['dir/.php-cs-fixer.cache', './dir3/file', './dir3/file'],
            ['dir/.php-cs-fixer.cache', 'dir3/file', 'dir3/file'],
```

And they will all pass. But I wonder if when the file starts with `./` we should expect the result without, so for example the first two cases would be
```php
            [null, './my/file', 'my/file'],
            [null, 'my/file', 'my/file'],
```